### DISCUSSION TITLE
Using checkver in architecture?

### DISCUSSION THREAD

**Discussion by amreus** [ScoopInstaller/Scoop#4641] [Help and Questions] [answered]
The following wiki page[1] section says `checkver` can be used in `architecture` but I'm not having any luck getting it to work. I scanned `checkver.ps1` and it doesn't appear to be supported. Have I missed something?

[1] https://github.com/ScoopInstaller/Scoop/wiki/App-Manifests#optional-properties

**Comment by HUMORCE** [comment, marked as answer]
an unimplemented feature.

**Comment by rashil2000** [comment]
https://github.com/ScoopInstaller/Scoop/issues/4611
